File: example/python/simple/example_2.py
from src import MdisClient


def main():
    client = None
    try:
        client = MdisClient.connect("127.0.0.1", 6411)
        my_token = client.get("token")
        print(f"resonse: {my_token}")

    except Exception as error:
        print(f"Error: {error}")

    finally:
        if client:
            client.close()


if __name__ == "__main__":
    main()

File: example/nodejs/simple/example_2.js
const MdisClient = require("./src/index");

async function main() {
  try {
    let m = MdisClient.connect((host = "127.0.0.1"), (port = 6411));
    const myToken = await m.get("token");
    console.log(myToken);
  } catch (error) {
    console.log(error);
  }
}
main();
